src/app/career-table/career-table.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { DataService } from '../_services/data.service';
+
+@Component({
+  selector: 'app-career-table',
+  templateUrl: './career-table.component.html',
+  styleUrls: ['./career-table.component.css']
+})
+export class CareerTableComponent implements OnInit {
+
+  rolesCapabilityMap = new Map<string, string[]>();
+  capabilities = [];
+  roleCapabilityData: any;
+  departmentName: any;
+
+  constructor(private dataService: DataService) { }
+ 
+  ngOnInit() {
+    //todo: pass in actual department id in user story 2
```

Comment:
It's usually a good idea to do this, but in agile projects, TODO comments can be lost and forgotten about so easily. I'd just remove this comment